src/light/bake.js

```diff
@@ -3,34 +3,39 @@ import callServices from '../utils/services/call.js'
 import whenDone from '../utils/processing/when-done.js'
 
 // main
+function bakeStage(stage) {
+  return function bake(storageId, options) {
+    // API
+    options = options || {}
 
-function bake(storageId, options) {
-  // API
-  options = options || {}
-  var sunDirection = sunDirection || [
-    0.7487416646324341,
-    -0.47789104947352223,
-    -0.45935396425474223
-  ]
+    // Optional bake parameters for API call
+    var bakeSettings = { sunDirection: options.sunDirection || [ 0.75, -0.48, -0.46 ] }
+    if (options.lightMapCount) { bakeSettings.lightMapCount = options.lightMapCount }
+    if (options.samples) { bakeSettings.samples = options.samples }
 
-  // internals
-  var assetStorageIds = []
-  // TODO: reimplement caching mechanism on server side
-  var cacheKey = null
+    // internals
+    // TODO: reimplement send "exportable" textures to bake
+    var assetStorageIds = []
+    // TODO: reimplement caching mechanism on server side
+    var cacheKey = null
 
-  return callServices('Processing.task.enqueue', {
-    method: 'bakePreview',
-    params: {
-      inputFileKey: storageId,
-      options: {
-        inputAssetKeys: assetStorageIds,
-        sunDirection: sunDirection,
-        cacheKey: cacheKey
+    var bakeParams = {
+      method: 'bake'.concat('.', stage),
+      params: {
+        inputFileKey: storageId,
+        //inputAssetKeys: assetStorageIds,
```

Comment:
did we remove this option or why is it commented out?

Reply:
it's currently an empty array hardcoded. i can remove it

Reply:
nah, was just wondering